scss/foundation_dependencies.scss

```diff
@@ -25,7 +25,6 @@
 
 
 // Imports
-@include foundation-typography;
 @include foundation-visibility-classes;
 
 {{#components}}
```

Comment:
The components are automatically rendered into the SASS file using the configuration file https://github.com/zurb/joyride/blob/master/gulp/config.js. Only Reveal and Tooltip are used here, plus some utility stuff.